lib/classes/TradeOffer.js

```diff
@@ -248,19 +255,22 @@ TradeOffer.prototype.removeTheirItems = function(items) {
 
 function addItem(details, offer, list) {
 	if (offer.id) {
-		throw new Error("Cannot add items to an already-sent offer");
+		throw new Error('Cannot add items to an already-sent offer');
 	}
 
-	if (typeof details.appid === 'undefined' || typeof details.contextid === 'undefined' || (typeof details.assetid === 'undefined' && typeof details.id === 'undefined')) {
-		throw new Error("Missing appid, contextid, or assetid parameter");
+	if (typeof details.appid === 'undefined'
+		|| typeof details.contextid === 'undefined'
+		|| (typeof details.assetid === 'undefined'
+			&& typeof details.id === 'undefined')) {
```

Comment:
I don't think long lines are good, but having them shouldn't be a problem especially if there's not much of them. I'm also a fan of helper functions/computed properties (`.needsConfirmation`). That's why dropping the strict limit and adding sounds like a good idea to me.

If you want to keep the limit, I'd suggest wider length. In my projects I use the indicator after 140th character (as a guideline, not strict limit). Don't forget the codebase is written in JS and the developers are hipsters writing on their 4k screens.